packages/vite/src/node/server/index.ts

```diff
@@ -267,6 +267,11 @@ export interface ViteDevServer {
    * @param forceOptimize - force the optimizer to re-bundle, same as --force cli flag
    */
   restart(forceOptimize?: boolean): Promise<void>
+
+  /**
+   * Open browser
+   */
+  openDevBrowser(): void
```

Comment:
Since this introduces a new public API, I'll put this on the discussion board, but my gut is that it's a nice API to have, maybe we should rename it as `openBrowser` instead, since it's already in the `ViteDevServer`.

Reply:
I think this is good. I also prefer to use openBrowser

Reply:
Yeah we just discussed this and decided to go with `openBrowser`

Reply:
OK, I'll adjust it